### PR TITLE
added security headers for more safety

### PR DESCRIPTION
**Pull request by sspooky13**
| Q             | A
| ------------- | ---
|Description, reason for the PR| added security headers in local development
|New feature| No
|[BC breaks](https://docs.shopsys.com/en/latest/contributing/backward-compatibility-promise/)| No
|Fixes issues| ...
|Have you read and signed our [License Agreement for contributions](https://www.shopsys.com/license-agreement)?| Yes

----
:globe_with_meridians: Live Preview:
  - https://pt-headers.odin.shopsys.cloud
  - https://cz.pt-headers.odin.shopsys.cloud

### CODE REVIEW

File: docker/nginx/nginx.conf [hunk 1]
```diff
@@ -33,6 +33,9 @@ server {
 
     add_header Access-Control-Allow-Origin "*" always;
     add_header Access-Control-Allow-Credentials "false" always;
+    add_header X-Frame-Options "SAMEORIGIN";
+    add_header X-Content-Type-Options "nosniff";
+    add_header X-XSS-Protection "1; mode=block";
```
Comment (grossmannmartin) [inline docker/nginx/nginx.conf]: according to https://securityheaders.cz/x-xss-protection and https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/X-XSS-Protection chrome, firefox, and edge don't use the XSS auditor, so I believe this header should not be implemented in favor of recommended CSP.
I don't insist on implementing CSP in this PR as it's not clear and easy task

File: UPGRADE-15.0.md [hunk 2]
```diff
@@ -87,6 +87,8 @@ Follow the instructions in relevant sections, e.g. `shopsys/coding-standards` or
     -   see #project-base-diff to update your project
 -   Added check vulnerabilities for javascripts in storefront ([#2993](https://github.com/shopsys/shopsys/pull/2993))
     -   see #project-base-diff to update your project
+-   added security headers for more safety in locale development ([#3050](https://github.com/shopsys/shopsys/pull/3050))
```
Comment (grossmannmartin) [inline UPGRADE-15.0.md]:
```suggestion
-   added security headers for more safety in local development ([#3050](https://github.com/shopsys/shopsys/pull/3050))
```

**Review by grossmannmartin — CHANGES_REQUESTED** [review]

File: docker/nginx/nginx.conf (hunk 1 above)
Comment (sspooky13) [reply docker/nginx/nginx.conf]: understand, I removed X-XSS-Protection. https://github.com/shopsys/shopsys/pull/3050/commits/6825ad1d988c04e27dfe9c2898a3d56c68c58322

File: UPGRADE-15.0.md (hunk 2 above)
Comment (sspooky13) [reply UPGRADE-15.0.md]: fixed https://github.com/shopsys/shopsys/pull/3050/commits/6825ad1d988c04e27dfe9c2898a3d56c68c58322

**Review by grossmannmartin — APPROVED** [review]

**Pull request merged**